crates/rspack_core/src/compiler/make/repair/build.rs

```diff
@@ -108,8 +106,23 @@ impl Task<MakeTaskContext> for BuildResultTask {
       build_result,
       diagnostics,
       current_profile,
+      plugin_driver,
+      compilation_id,
     } = *self;
 
+    module.set_build_info(build_result.build_info);
+    module.set_build_meta(build_result.build_meta);
+
+    plugin_driver
+      .compilation_hooks
+      .succeed_module
+      .call(compilation_id, &mut module)
+      .await?;
+
+    let build_info_default = BuildInfo::default();
+
+    let build_info = module.build_info().unwrap_or(&build_info_default);
```

Comment:
use `unwrap_or_default`

Reply:
we can't use `unwrap_or_default `, due to `Default` Trait is not implement for &BuildInfo even though implement  for BuildInfo